app/Views/Templates/components/elements/avatar.blade.php

```diff
@@ -0,0 +1,37 @@
+@props([
+    'src' => '',                 // URL for the avatar image
+    'alt' => 'Avatar',           // Alt text for the avatar image
+    'size' => 'w-12 h-12',       // Size of the avatar (default is medium size)
```

Comment:
Size should not just be overwritten. Instead we should pass in semantic sizing like small, medium large and the component decides which classses to choose